### ISSUE TITLE
...\\pack\\packer\\start\\mason.nvim/lua/mason-core/receipt.lua:104: primary_source is required

### ISSUE THREAD

**Issue by re-kaptcha** [williamboman/mason.nvim#236]
### I've made an initial investigation into the error myself

- [X] Yes

### I've recently downloaded the latest plugin version of mason.nvim

- [X] Yes

### Problem description

When installing the taplo language server, I get the following error:
```
...\\pack\\packer\\start\\mason.nvim/lua/mason-core/receipt.lua:104: primary_source is required
```

### Neovim version (>= 0.7)

NVIM v0.8.0-dev-768-gb8dcbcc73
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compiled by runneradmin@fv-az8-588

### Operating system/version

Windows 11 Version 10.0.22000 Build 22000

### Affected packages

Taplo

### Actual behavior

`...\\pack\\packer\\start\\mason.nvim/lua/mason-core/receipt.lua:104: primary_source is required` when trying to install Taplo

### Expected behavior

Install with no errors like other LSPs

### Mason output

```Text
mason.nvim 
                                press ? for help
                   https://github.com/williamboman/mason.nvim
   (1) All   (2) LSP   (3) DAP   (4) Linter   (5) Formatter  
  
  Language Filter: press <C-f> to apply filter

  Installed
    ◍ editorconfig-checker
    ◍ stylua
    ◍ yamllint
    ◍ prettierd
    ◍ pylint
    ◍ csharpier
    ◍ flake8
    ◍ eslint_d
    ◍ pyright
    ◍ stylelint-lsp
    ◍ yaml-language-server
    ◍ typescript-language-server
    ◍ vim-language-server
    ◍ json-lsp
    ◍ powershell-editor-services
    ◍ rust-analyzer
    ◍ html-lsp
    ◍ omnisharp
    ◍ lua-language-server
    ◍ diagnostic-languageserver
    ◍ emmet-ls
    ◍ css-lsp
    ◍ dockerfile-language-server
    ◍ bash-language-server
    ◍ clangd
  
  Failed
    ◍ taplo
      ...\pack\packer\start\mason.nvim/lua/mason-core/receipt.lua:104: primary_source is required
```

### Installation log

```Text
[INFO  8/3/2022 3:55:33 PM] ...acker\start\mason.nvim/lua/mason-core/installer/init.lua:110: Executing installer for Package(name=taplo)
[DEBUG 8/3/2022 3:55:34 PM] ...\site\pack\packer\start\mason.nvim/lua/mason-core/fs.lua:70: fs: mkdirp C:\Users\{USER}\AppData\Local\nvim-data\mason\.packages\taplo
[DEBUG 8/3/2022 3:55:34 PM] ...er\start\mason.nvim/lua/mason-core/installer/context.lua:174: Promoting cwd "C:\\Users\\{USER}\\AppData\\Local\\nvim-data\\mason\\.packages\\taplo" to "C:\\Users\\{USER}\\AppData\\Local\\nvim-data\\mason\\packages\\taplo"
[DEBUG 8/3/2022 3:55:34 PM] ...\site\pack\packer\start\mason.nvim/lua/mason-core/fs.lua:83: fs: rename C:\Users\{USER}\AppData\Local\nvim-data\mason\.packages\taplo C:\Users\{USER}\AppData\Local\nvim-data\mason\packages\taplo
[DEBUG 8/3/2022 3:55:34 PM] ...ker\start\mason.nvim/lua/mason-core/installer/linker.lua:80: Linking Package(name=taplo)
[DEBUG 8/3/2022 3:55:34 PM] ...acker\start\mason.nvim/lua/mason-core/installer/init.lua:30: Writing receipt for Package(name=taplo)
[ERROR 8/3/2022 3:55:34 PM] ...acker\start\mason.nvim/lua/mason-core/installer/init.lua:146: Installation failed for Package(name=taplo) error="...\\pack\\packer\\start\\mason.nvim/lua/mason-core/receipt.lua:104: primary_source is required"
[DEBUG 8/3/2022 3:55:34 PM] ...\site\pack\packer\start\mason.nvim/lua/mason-core/fs.lua:46: fs: rmrf C:\Users\{USER}\AppData\Local\nvim-data\mason\packages\taplo
[DEBUG 8/3/2022 3:55:34 PM] ...ker\start\mason.nvim/lua/mason-core/installer/linker.lua:21: Unlinking Package(name=taplo)
```

### Healthcheck

```Text
mason: require("mason.health").check()
========================================================================
## mason.nvim report
  - OK: neovim version >= 0.7.0
  - OK: **Go**: `go version go1.19 windows/amd64`
  - OK: **cargo**: `cargo 1.62.1 (a748cf5a3 2022-06-08)`
  - WARNING: **luarocks**: not available
  - WARNING: **Ruby**: not available
  - WARNING: **RubyGem**: not available
  - WARNING: **Composer**: not available
  - WARNING: **PHP**: not available
  - OK: **npm**: `8.11.0`
  - OK: **node**: `v16.16.0
`
  - WARNING: **python3**: not available
  - WARNING: **pip3**: not available
  - WARNING: **javac**: not available
  - WARNING: **java**: not available
  - WARNING: **julia**: not available
  - ERROR: **wget**: not available
  - OK: **curl**: `curl 7.83.1 (Windows) libcurl/7.83.1 Schannel
`
  - ERROR: **gzip**: not available
  - OK: **tar**: `bsdtar 3.5.2 - libarchive 3.5.2 zlib/1.2.5.f-ipp bz2lib/1.0.6 
`
  - OK: **python**: `Ok`
  - OK: **pip**: `pip 22.1.2 from C:\Users\{USER}\AppData\Local\Programs\Python\Python310\lib\site-packages\pip (python 3.10)

`
  - OK: GitHub API rate limit. Used: 6. Remaining: 4994. Limit: 5000. Reset: 8/3/2022 4:45:30 PM.
```

### Screenshots

![image](https://user-images.githubusercontent.com/50426223/182697927-7e98016e-358f-40b5-aa33-f825fc50a53e.png)
![image](https://user-images.githubusercontent.com/50426223/182698658-b4ae3f31-66e6-43cc-84c5-efd0c00d832f.png)
(It doesn't seem to be caused by `mason-tool-installer`, as the error references a `mason.nvim` file)

**Comment by williamboman** [comment]
Hey! Should be fixed in #237

**Comment by re-kaptcha** [comment]
Yep! Seems to be installing now. Thanks!
